models/reservation.js

```diff
@@ -50,23 +46,15 @@ const reservationSchema = new mongoose.Schema(
   { timestamps: true }
 )
 
-const Reservation = mongoose.model('Reservation', reservationSchema)
-
-const validateReservation = (reservation) => {
-  const schema = Joi.object({
-    userId: Joi.objectId().required(),
-    hotelId: Joi.objectId().required(),
-    roomId: Joi.objectId().required(),
-    startDate: Joi.date().required(),
-    endDate: Joi.date().required(),
-    people: {
-      children: Joi.number().min(0).required(),
-      adults: Joi.number().min(1).required(),
-    },
-  })
+reservationSchema.methods.toJSON = function () {
+  var obj = this.toObject()
+  delete obj.createdAt
+  delete obj.updatedAt
```

Comment:
if we delete them from the response we should leave it so we can have them in logs